cfpb/cfgov-refresh
# Fix for your-story/tell-your-story link

Fix for your-story/tell-your-story link


## Changes

- Modifed `cfgov/jinja2/v1/your-story/index.html` to make `https://story.consumerfinance.gov/tell-your-story/s/tell-your-story` and absolute reference.

## Testing

1. Visit `/your-story/`
2. Click the `Tell your story` buttons.
2. You should be taken to https://story.consumerfinance.gov/tell-your-story/s/tell-your-story.

## Checklist
